packages/babel-parser/src/parser/expression.js

```diff
@@ -1120,6 +1120,17 @@ export default class ExpressionParser extends LValParser {
           );
         }
 
+        if (
+          this.match(tt.comma) &&
+          (this.lookahead().type === tt.name ||
+            this.lookahead().type === tt.ellipsis)
+        ) {
+          this.raise(
+            this.state.start,
+            "Rest parameter must be last formal parameter",
```

Comment:
Thanks for your PR, I looks good.

At first the error mesage looked odd to me, but it's actually V8's one :+1:

Reply:
@xtuc thanks! I'm glad that my first PR was not bad 😃 Yeah, I decided to get this message from V8.